ksql-common/src/main/java/io/confluent/ksql/util/StringUtil.java

```diff
@@ -26,12 +26,13 @@ private StringUtil() {
   public static String cleanQuotes(final String stringWithQuotes) {
     // TODO: move check to grammar
     if (stringWithQuotes.startsWith("'") && stringWithQuotes.endsWith("'")) {
-      return stringWithQuotes.substring(1, stringWithQuotes.length() - 1);
+      return stringWithQuotes.substring(1, stringWithQuotes.length() - 1)
+                .replaceAll("'(.*)'", "$1");
     }
-    return stringWithQuotes;
+    return stringWithQuotes.replaceAll("'(.*)'", "$1");
```

Comment:
I would not change the string if the passed in string is not actually quoted, i.e. I would just return `stringWithQuotes`. 

I would also suggest changing the replace to just replace all double single-quotes with a single-quote, i.e.  `.replaceAll("''", "')`.

Making this method:

```
public static String cleanQuotes(final String stringWithQuotes) {
    // TODO: move check to grammar
    if (!stringWithQuotes.startsWith("'") || !stringWithQuotes.endsWith("'")) {
      return stringWithQuotes;
    }

    return stringWithQuotes
              .substring(1, stringWithQuotes.length() - 1)
              .replaceAll("''", "'");
}
```

Reply:
But,  what happen if it receives something like:
`yyyy-MM-dd''T''HH:mm:ssX`

should  the code return `yyyy-MM-dd'T'HH:mm:ssX`?

I don't know, 😄maybe it's impossible to input this String